apps/condo/domains/organization/schema/AcceptOrRejectOrganizationEmployeeRequestService.js

```diff
@@ -0,0 +1,189 @@
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, getByCondition, getById } = require('@open-condo/keystone/schema')
+
+const { COMMON_ERRORS } = require('@condo/domains/common/constants/errors')
+const access = require('@condo/domains/organization/access/AcceptOrRejectOrganizationEmployeeRequestService')
+const { OrganizationEmployeeRequest, OrganizationEmployee } = require('@condo/domains/organization/utils/serverSchema')
+
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    REQUEST_NEEDS_ACCEPT_OR_REJECT: {
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+        variable: ['data'],
+        code: BAD_USER_INPUT,
+        type: 'REQUEST_NEEDS_ACCEPT_OR_REJECT',
+        message: 'The request must be accepted or rejected',
+    },
+    REQUEST_ALREADY_DECIDED: {
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+        variable: ['data', 'employeeRequest'],
+        code: BAD_USER_INPUT,
+        type: 'REQUEST_ALREADY_DECIDED',
+        message: 'The request has already been decided',
+    },
+    REQUEST_NOT_FOUND: {
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+        variable: ['data', 'employeeRequest'],
+        code: BAD_USER_INPUT,
+        type: 'REQUEST_NOT_FOUND',
+        message: 'The request not found',
+    },
+    EMPLOYEE_ROLE_REQUIRED: {
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+        variable: ['data', 'employeeData', 'role'],
+        code: BAD_USER_INPUT,
+        type: 'EMPLOYEE_ROLE_REQUIRED',
+        message: 'Employee role required',
+    },
+    USER_NOT_FOUND: {
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+        variable: ['data', 'employeeRequest'],
+        code: BAD_USER_INPUT,
+        type: 'USER_NOT_FOUND',
+        message: 'No user found to join the organization',
+    },
+    USER_DOES_NOT_HAVE_PHONE: {
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+        variable: ['data', 'employeeRequest'],
+        code: BAD_USER_INPUT,
+        type: 'USER_DOES_NOT_HAVE_PHONE',
+        message: 'The user does not have a phone',
+    },
+    DV_VERSION_MISMATCH: {
+        ...COMMON_ERRORS.DV_VERSION_MISMATCH,
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+    },
+    WRONG_SENDER_FORMAT: {
+        ...COMMON_ERRORS.WRONG_SENDER_FORMAT,
+        mutation: 'acceptOrRejectOrganizationEmployeeRequest',
+    },
+}
+
+const AcceptOrRejectOrganizationEmployeeRequestService = new GQLCustomSchema('AcceptOrRejectOrganizationEmployeeRequestService', {
+    types: [
+        {
+            access: true,
+            type: 'input AcceptOrRejectOrganizationEmployeeRequestEmployeeDataInput { role: OrganizationEmployeeRoleWhereUniqueInput! }',
```

Comment:
If it is necessary to specify other data when accepting an employee into the organization, it will be more convenient to do this.
For now this is `role`, but `OrganizationEmployee` has other fields